pyquil/tests/conftest.py

```diff
@@ -22,7 +26,7 @@ def isa_dict():
             "1-2": {
                 "type": "ISWAP"
             },
-            "2-0": {
```

Comment:
?

Reply:
This was a malformed ISA. `quilc` complains if the indices are not ordered when specifying edges